Core/src/org/sleuthkit/autopsy/datamodel/BlackboardArtifactNode.java

```diff
@@ -155,11 +165,11 @@ public BlackboardArtifactNode(BlackboardArtifact artifact) {
         this.setName(Long.toString(artifact.getArtifactID()));
         this.setDisplayName();
         this.setIconBaseWithExtension(ExtractedContent.getIconFilePath(artifact.getArtifactTypeID())); //NON-NLS
-        Case.addPropertyChangeListener(pcl);
+        Case.addEventSubscriber(CASE_EVENTS_OF_INTEREST, WeakListeners.propertyChange(pcl, null));
```

Comment:
Could we modify Case.addEventSubscriber to wrap the listenerers in the WeakListener "proxy" automatically?  Then this wouldn't crop up again/in other places.